wnfs-wasm/tests/private.spec.ts

```diff
@@ -141,15 +129,10 @@ test.describe("PrivateDirectory", () => {
         new Date(),
         forest,
         store,
-        rng
+        rng,
       );
 
-      var result = await rootDir.getNode(
-        ["pictures", "cats", "tabby.png"],
-        true,
-        forest,
-        store
-      );
+      var result = await rootDir.getNode(["pictures", "cats", "tabby.png"], true, forest, store);
```

Comment:
Just now noticed that my editor the auto formatting thing again 🤦 Let me know if I should undo this.

Reply:
It's fine. Don't worry about it :v: